models/api.models.js

```diff
@@ -23,7 +23,16 @@ exports.retrieveArticle = async (id) => {
   }
 };
 
-exports.retrieveAllArticles = async () => {
+exports.retrieveAllArticles = async (sort_by='created_at', order='DESC') => {
+  const dataColumns = ["article_id","title","topic","author", "created_at", "votes", "article_img_url"]
+
+  if (!dataColumns.includes(sort_by)) {
+    return Promise.reject({status: 404, msg: "sort_by Not Found"})
+  }
+  if (order.toUpperCase() !== "DESC" && order.toUpperCase() !== "ASC") {
+    return Promise.reject({status: 404, msg: "order Not Found"})
+  }
```

Comment:
The error message for invalid queries ('sort_by Not Found' and 'order Not Found') could be improved to make it more descriptive, e.g., 'Invalid sort_by column' or 'Invalid order value'.